cmake/sysbuild/suit.cmake

```diff
@@ -368,6 +368,34 @@ function(suit_create_package)
     endif()
   endif()
 
+  if(SB_CONFIG_SUIT_MULTI_IMAGE_PACKAGE_BUILD)
+    include(${ZEPHYR_BASE}/../nrf/cmake/fw_zip.cmake)
```

Comment:
please use `${ZEPHYR_NRF_MODULE_DIR}`

Reply:
Can I do it in the next PR as a follow-up? I would like to merge it to unblock other work.